cocotb/share/def/modelsim.def

```diff
@@ -100,3 +100,16 @@ mti_TickLeft
 mti_TickLength
 mti_TickRight
 mti_VsimFree
+mti_Interp
+mti_Cmd
+Tcl_GetStringResult
+Tcl_ResetResult
+Tcl_GetObjResult
+Tcl_ResetResult
+Tcl_ListObjGetElements
+Tcl_GetStringResult
+TclFreeObj
+Tcl_ResetResult
+Tcl_ResetResult
+Tcl_GetString
+TclFreeObj
```

Comment:
Without these lines, modelsim is unusable on windows, right? Or is that true of this whole PR anyway?

Reply:
Without those lines, it will not install. 

It Will not stimulate without the rest.

Reply:
To be clear will not install with FLI support.